custom-metrics-stackdriver-adapter/adapter.go

```diff
@@ -211,7 +228,13 @@ func main() {
 		"Stackdriver Endpoint used by adapter. Default is https://monitoring.googleapis.com/")
 	flags.BoolVar(&serverOptions.EnableDistributionSupport, "enable-distribution-support", serverOptions.EnableDistributionSupport,
 		"enables support for scaling based on distribution values")
-
+	// Custom flags
+	flags.StringVar(&serverOptions.ProjectId, "project-id", "",
+		"Project ID of the Google Cloud cluster. May be left empty on GKE.")
```

Comment:
This is the project ID where the stackdriver (a.k.a. google managed prometheus) instance is right? doesn't need to be the same project ID as the currently running cluster right?

Reply:
I thought the Project ID concept was a Google-Cloud only term? I thought we're using this for authentication or validation with GPC where we're storing the data. Is that not clear or incorrect?

Reply:
That is correct, but I don't think that's quite what this description says. To me "cluster" refers to a k8s cluster, which may be in a different GCP project or even different cloud than the GCP project in which the metrics are stored.

Reply:
Oh, you'd prefer something like "Project ID on Google Cloud" that doesn't mention Cluster?